packages/connectivity/src/scp-cf/destination/destination-from-service.ts

```diff
@@ -273,7 +273,7 @@ class DestinationFromServiceRetriever {
       );
     }
     if (credentials.length > 1) {
-      logger.info(
+      logger.debug(
```

Comment:
I guess if there are multiple destination service instance subscribed it will not hurt in general all will return the subaccount destinations. It is indicating some misconfiguration in most cases so putting it to warn would be also fine to indicate the random behavior. 

=> In favor for warn.